backend/core/utils.py

```diff
@@ -64,7 +64,11 @@ def user(self):
     def user(self, user):
         self._user = user
     
-    def detailView(self, request):
+    def detailView(self, request, *args):
         result = self._user.getTemplatesLocation() + "detail.html"
-        data = {'user': self._user}
+        print()
+        data = {
+            'user': self._user,
+            'assessment' : args[0],
```

Comment:
Está gerando o erro "tuple index out of range" nesse args[0] na view detailClient

Reply:
OK